packages/ui/src/components/labels/CountWithIcon.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import '@ferlab/style/components/labels/CountWithIcon.scss'
+
+export enum CountWithIconTypeEnum  {
+    Inline = 'inline',
+    Stack = 'stack',
+}
+
+export type ICountWithIconProps = {
+    className?: string;
+    Icon: React.ReactNode;
+    type?: CountWithIconTypeEnum;
+    label: string;
+    total: number | string;
```

Comment:
We should accept React.Node here since we removed the translation from the original component it will be used with React.intl and the translation component returns a component not a string